docs/Getting-Started/Installation/installation.md

```diff
@@ -6,7 +6,7 @@ The Pop!_OS installer guides the user through setup of basic system settings.
 - Choose a locale and keyboard language.
 - Perform system hard drive selection and partitioning (advanced).
 - Create a user account.
-- Enable system disk encryptions.
+- Enable system disk encryption.
 
 ---
 
```

Comment:
A link here of how to disable secure boot might be useful.

https://itsfoss.com/disable-secure-boot-windows/

Reply:
Not commenting on whether more detailed steps or a link should be added, but this article initially had a link to official Microsoft documentation and it was discussed here: https://github.com/pop-os/docs/pull/2#discussion_r840029648

Reply:
I see, that makes sense. Thanks for the info.

Reply:
I'd still prefer to leave this out for now. Since these are machine-specific instructions, I feel readers are better served by the abundance of external articles on this topic. We can always add this later.